marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/writer/WriteJobWithCustomEndpointsTest.java

```diff
@@ -38,7 +38,10 @@ void bothEndpointsAreCustom() {
 
         verifyCustomInitializeEndpointIsUsed();
         dataSourceWriter.commit(null);
-        verifyCustomFinalizeEndpointIsUsed();
+        assertEquals("stop-on-error", getJobDocumentStatus(),
```

Comment:
Just to avoid duplication, toss this into a new private method - e.g verifyStatusIsStopOnError.